server.js

```diff
@@ -9,7 +9,7 @@ const webpackDevMiddleware = require("webpack-dev-middleware");
 const app = express();
 const compiler = webpack(config);
 
-app.use("/js", express.static("js"));
```

Comment:
Why'd you change this? Any specific reason for the storybook? I personally like the URL matching the filesystem, and we may add other directories later.